package.json

```diff
@@ -82,7 +82,7 @@
     "sqs-consumer": "^3.6.1",
     "style-loader": "^0.18.2",
     "sweetalert": "^1.1.3",
-    "tedious": "^2.0.0",
+    "tedious": "^2.7.1",
```

Comment:
Needed to upgrade the library because when calling close if we hadn't made a request, the library was throwing an exception.  Looked at the changelog, this is the latest version which didn't have breaking changes